python/pyspark/mllib/linalg.py

```diff
@@ -778,6 +778,8 @@ def dense(elements):
         >>> Vectors.dense([1, 2, 3])
         DenseVector([1.0, 2.0, 3.0])
         """
+        if isinstance(elements, (int, float)):
```

Comment:
This conflicts with / overlaps with the changes in #7476, so I guess this patch is blocking until that other one gets reviewed?